libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -50,6 +50,9 @@
 
 #define INTERNAL_TEMPERATURE_CLAMP 35.0f
 
+#define BARO_LOOP_FREQ    10.f                  // GPS loop frequency in Hz
```

Comment:
The comment is inconsistent (talks about GPS on a baro define), and expecting the GPS update rate to not exceed 10 Hz is an unsafe assumption, especially to put in a define like this.

Reply:
Agree, will reconsider